components/definitions/rewardsExtensions.d.ts

```diff
@@ -48,40 +46,45 @@ declare namespace RewardsExtension {
     status?: PublisherStatus
   }
 
-  export interface Grant {
-    altcurrency: string
-    probi: string
-    expiryTime: number
-    type: string
+  export type CaptchaStatus = 'start' | 'wrongPosition' | 'generalError' | 'finished' | null
+
+  export enum PromotionTypes {
+    UGP = 0,
+    ADS = 1
   }
 
-  export type GrantStatus = 'wrongPosition' | 'grantGone' | 'generalError' | 'grantAlreadyClaimed' | number | null
+  export enum PromotionStatus {
+    ACTIVE = 0,
+    ATTESTED = 1,
+    CLAIMED = 2,
+    SIGNED_TOKENS = 3,
+    FINISHED = 4,
+    OVER = 5
+  }
```

Comment:
`PromotionTypes` and `PromotionStatus` are defined in `rewards.d.ts` - is there any way to put them into a common 'include' in ts?

Reply:
we need to work on this as we duplicate a lot of definitions right now. Please log an issue for it if we don't already have it

Reply:
https://github.com/brave/brave-browser/issues/6954